apps/chat/src/components/Header/User/UserMobile.tsx

```diff
@@ -19,11 +20,24 @@ import LogOutIcon from '../../../../public/images/icons/log-out.svg';
 import UserIcon from '../../../../public/images/icons/user.svg';
 
 const UserInfo = () => {
+  const { t } = useTranslation(Translation.SideBar);
+
   const { data: session } = useSession();
+
   return (
-    <div className=" w-full border-b border-tertiary p-2 text-primary">
+    <div className="w-full border-b border-tertiary p-2 text-primary">
       <div className="flex h-[42px] items-center">
-        <UserIcon className="mx-2" width={18} height={18} />
+        {session?.user?.image ? (
+          <Image
```

Comment:
There was some issues with `next\image` that why we are using `img` in other places (@mikitabut can know details)

Reply:
There were some problems with basePath. Reverted to <img ... />